cmd/libs/go2idl/conversion-gen/generators/conversion.go

```diff
@@ -46,13 +46,18 @@ type CustomArgs struct {
 	SkipUnsafe bool
 }
 
-// This is the comment tag that carries parameters for conversion generation.
+// There are the comment tags that carry parameters for conversion generation.
```

Comment:
s/There/These/

Document what argument these tag names accept and its exact format.